app.js

```diff
@@ -135,19 +136,30 @@ const styles = StyleSheet.create({
   },
 })
 
+let settingsButtonActive = false
+
+function openSettings(route, navigator) {
+  settingsButtonActive = true
+  navigator.push({
+    id: 'SettingsView',
+    title: 'Settings',
+    index: route.index + 1,
+    sceneConfig: Navigator.SceneConfigs.FloatFromBottom,
+  })
+}
+
+function closeSettings(navigator) {
+  navigator.pop()
+  settingsButtonActive = false
```

Comment:
and should we flip these two?

I don't think there'll be any functional difference